.env.sample

```diff
@@ -118,9 +120,19 @@ CLUSTER_METADATA_MONGODB_PORT=27018
 CLUSTER_METADATA_MONGO_EXPRESS_PORT=8082
 CLUSTER_METADATA_SERVICE_CLIENT_SECRET=oLQhekWsZ4n6OYL
 CLUSTER_METADATA_SERVICE_CLUSTER_ACTIVITY_WINDOW_MILLIS=3600000
+CLUSTER_METADATA_APPLICATION_TOPIC=ApplicationMetadataUpdated
+CLUSTER_METADATA_NODE_TOPIC=NodeMetadataUpdated
```

Comment:
Consider moving topics to the part with message broker as they will be shared by many services